pastis/temporal_analysis/mid_zernike_analysis.py

```diff
@@ -0,0 +1,72 @@
+import os
```

Comment:
I just saw this new file because the flake8 failed on it. This is a literal copy-paste of a notebook - let's not do it this way, please. I think it would be best if you delete it. Please remember, the purpose of this PR is to get the work from the notebooks merged - we will work on getting them integrated into scripts separately.